### PR TITLE
Update README to include high level features

### PR DESCRIPTION
**Pull request by riggaroo**
Including high level feature list pointing to different resources for the README, to enable easier navigation of the samples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,17 +42,50 @@ The branch `compose-latest` is targetting the latest alpha versions of Compose.
...
+
+## High level features 
+
+Looking for a sample that has the following features?
+
+### Custom Layouts
+* [Jetnews: Interests Screen](https://github.com/android/compose-samples/blob/ee198110d8a7575da281de9bd0f84e91970468ca/JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt#L428)
+* [Jetchat: AnimatedFabContent](https://github.com/android/compose-samples/blob/ee198110d8a7575da281de9bd0f84e91970468ca/Jetchat/app/src/main/java/com/example/compose/jetchat/components/AnimatingFabContent.kt#L101)
+* [Jetsnack: Grid](https://github.com/android/compose-samples/blob/73d7f25815e6936e0e815ce975905a6f10744c36/Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Grid.kt#L27)
```
Comment (bentrengrove) [inline README.md:60]: suggestion: Another one you could include is

* [Jetsnack: CollapsingImageLayout](https://github.com/android/compose-samples/blob/main/Jetsnack/app/src/main/java/com/example/jetsnack/ui/snackdetail/SnackDetail.kt)

File: README.md [hunk 2]
```diff
@@ -42,17 +42,50 @@ The branch `compose-latest` is targetting the latest alpha versions of Compose.
...
+* [Owl: Transition - Topic Chip change](https://github.com/android/compose-samples/blob/5dd149f2a5d35527cb628972c6c058057233647f/Owl/app/src/main/java/com/example/owl/ui/onboarding/Onboarding.kt#L181)
+* [Jetsnack: Parallax Scroll](https://github.com/android/compose-samples/blob/e9f5d03ccc7de29165853914749fcf7af70de823/Jetsnack/app/src/main/java/com/example/jetsnack/ui/snackdetail/SnackDetail.kt)
+* [Owl: Bottom Sheet transforming](https://github.com/android/compose-samples/blob/5dd149f2a5d35527cb628972c6c058057233647f/Owl/app/src/main/java/com/example/owl/ui/course/CourseDetails.kt)
+
+### Text
+* [Jetchat: Downloadable Fonts](https://github.com/android/compose-samples/pull/787)
+
+### Large Screens
+* [JetCaster - Tabletop mode](https://github.com/android/compose-samples/blob/0f7d5958c57a83ecad10136da4d359ae07046d07/Jetcaster/app/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt#L138)
+* [JetNews - Window Size Classes](https://github.com/android/compose-samples/blob/69e9d862b5ffb321064364d7883e859db6daeccd/JetNews/app/src/main/java/com/example/jetnews/ui/MainActivity.kt#L36)
```
Comment (bentrengrove) [inline README.md:82]: nit: Inconsistent casing of JetCaster and JetNews here.

File: README.md [hunk 3]
```diff
@@ -42,17 +42,50 @@ The branch `compose-latest` is targetting the latest alpha versions of Compose.
...
+* [Jetsnack: Custom Design System](https://github.com/android/compose-samples/blob/main/Jetsnack/app/src/main/java/com/example/jetsnack/ui/theme/Theme.kt)
+
+### Animations
+* [Crane: Calendar Selection Animation](https://github.com/android/compose-samples/blob/87ab51822cd87ce07b8ea3ec9dcb8b93f04c0b2a/Crane/app/src/main/java/androidx/compose/samples/crane/calendar/Calendar.kt#L64)
+* [Crane: AnimatedContent](https://github.com/android/compose-samples/pull/843)
+* [Jetsurvey: AnimatedContent](https://github.com/android/compose-samples/pull/842)
+* [Jetcaster: Animated theme colors](https://github.com/android/compose-samples/blob/69e9d862b5ffb321064364d7883e859db6daeccd/Jetcaster/app/src/main/java/com/example/jetcaster/util/DynamicTheming.kt)
+* [Owl: Transition - Topic Chip change](https://github.com/android/compose-samples/blob/5dd149f2a5d35527cb628972c6c058057233647f/Owl/app/src/main/java/com/example/owl/ui/onboarding/Onboarding.kt#L181)
+* [Jetsnack: Parallax Scroll](https://github.com/android/compose-samples/blob/e9f5d03ccc7de29165853914749fcf7af70de823/Jetsnack/app/src/main/java/com/example/jetsnack/ui/snackdetail/SnackDetail.kt)
```
Comment (bentrengrove) [inline README.md:74]: Just saw this after leaving the comment above. The Jetsnack readme calls this a CollapsingImageLayout. Might make more sense in the custom layouts section but up to you.

Jetsnack also has a custom animating bottom bar, you could replace this one with that. https://github.com/android/compose-samples/blob/main/Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt

**Review by bentrengrove — APPROVED** [review]
Just some suggestions

File: README.md (hunk 3 above)
Comment (riggaroo) [reply README.md:75]:
```suggestion
* [Jetsnack: Animating Bottom Barl](https://github.com/android/compose-samples/blob/main/Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt)
```

File: README.md [hunk 4]
```diff
@@ -42,17 +42,51 @@ The branch `compose-latest` is targetting the latest alpha versions of Compose.
...
+* [Owl: Transition - Topic Chip change](https://github.com/android/compose-samples/blob/5dd149f2a5d35527cb628972c6c058057233647f/Owl/app/src/main/java/com/example/owl/ui/onboarding/Onboarding.kt#L181)
+* [Jetsnack: Animating Bottom Barl](https://github.com/android/compose-samples/blob/main/Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt)
+* [Owl: Bottom Sheet transforming](https://github.com/android/compose-samples/blob/5dd149f2a5d35527cb628972c6c058057233647f/Owl/app/src/main/java/com/example/owl/ui/course/CourseDetails.kt)
+
+### Text
+* [Jetchat: Downloadable Fonts](https://github.com/android/compose-samples/pull/787)
+
+### Large Screens
+* [JetCaster - Tabletop mode](https://github.com/android/compose-samples/blob/0f7d5958c57a83ecad10136da4d359ae07046d07/Jetcaster/app/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt#L138)
+* [JetNews - Window Size Classes](https://github.com/android/compose-samples/blob/69e9d862b5ffb321064364d7883e859db6daeccd/JetNews/app/src/main/java/com/example/jetnews/ui/MainActivity.kt#L36)
```
Comment (riggaroo) [inline README.md:83]:
```suggestion
* [Jetcaster - Tabletop mode](https://github.com/android/compose-samples/blob/0f7d5958c57a83ecad10136da4d359ae07046d07/Jetcaster/app/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt#L138)
* [Jetnews - Window Size Classes](https://github.com/android/compose-samples/blob/69e9d862b5ffb321064364d7883e859db6daeccd/JetNews/app/src/main/java/com/example/jetnews/ui/MainActivity.kt#L36)
```

**Pull request merged**